src/current/_includes/v23.1/ui/statement-details.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
This is not part of this PR, but if so inclined, there is a rendering issue on the Statements page under "Insights" where the markdown bullets are not rendering as bullet points.